.pipelines/rp-full-dev-setup.yml

```diff
@@ -0,0 +1,81 @@
+trigger: none
+pr: none
+
+parameters:
+  - template: ./rp-dev/rp-dev-params.yml
+
+resources:
+  containers:
+    - container: container
+      image: registry.access.redhat.com/ubi8/toolbox:8.8
+      options: --user=0 --privileged -v /dev/shm:/dev/shm --device /dev/net/tun --name vpn
+
+variables:
+  # - template: vars.yml # probably redundant
+  - template: ./rp-dev/rp-dev-vars.yml
+
+jobs:
+  - job: Full_RP_dev_setup_deployment
+    timeoutInMinutes: 180
+    pool:
+      name: 1es-aro-ci-pool
+
+    steps:
+# setup
+      - template: ./templates/template-checkout.yml
+      - template: ./templates/template-az-cli-login.yml # does it have az CLI alraedy?
+        parameters:
+          azureDevOpsJSONSPN: ${{ parameters.azureJson }}
+      - template: ./templates/rp-dev/template-download-secrets.yml
+        parameters:
+          secretSA: ${{ parameters.secretSA }}
+      - template: ./templates/rp-dev/template-generate-rp-dev-config.yml
+        parameters:
+          location: ${{ parameters.location }}
+          resourceGroup: ${{ parameters.resourceGroup }}
+          databaseAccountName: ${{ parameters.databaseAccountName }}
+          keyvaultPrefix: ${{ parameters.keyvaultPrefix }}
+          aroImage: ${{ parameters.aroImage }}
+      - script: make deploy # use different target which will succeed and won't fail
+        name: semi-deploy-aro-rp-dev
+        displayName: ⚙️ Run pre-deplyments tasks prior to AKS
+      - script: source ./hack/devtools/deploy-shared-env.sh && deploy_vpn_for_dedicated_rp
+        name: deploy-vpn-gateway
+        displayName: ⚙️ Deploy VPN Gateway
```

Comment:
q: what's the expected way for a developer to get the certificates to connect to the VPN resource?

Reply:
The developer may use his Azure account to log in and search the certificates in the Azure subscription, but it might be trickier than that.